### DISCUSSION TITLE
Logging Tracking

### DISCUSSION THREAD

**Discussion by portosricos** [kimai/kimai#3219] [Questions & Answers]
I would like to know if kimai has the possibility to record browser information with ip address for login/logout?

Thank you

**Comment by kevinpapst** [comment]
The Kiosk plugin (see Marketplace) has such a feature. The normal community version doesn't.

**Comment by portosricos** [comment]
I looked in the demo-plugin and I don't see the feature. 

To see the login of an employee and see the ipaddress of the devise he log in.
